File: back-end/ReportingTasksPhp/Models/ActualHours.php
<?php

class ActualHours implements JsonSerializable {

    public $ActualHoursId;
    public $UserId;
    public $ProjectId;
    public $CountHours;
    public $date;
    public function __construct($sqlRaw_) {
        $this->ActualHoursId = $sqlRaw_['actual_hours_id'];
        $this->UserId = $sqlRaw_['user_id'];
        $this->ProjectId = $sqlRaw_['project_id'];
        $this->CountHours = $sqlRaw_['count_houers'];
        $this->date = $sqlRaw_['date'];
    }
    public function jsonSerialize() {
        return get_object_vars($this);
    }

}

File: back-end/ReportingTasksProject/01_BOL/ActualHours.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BOL
{
  public  class ActualHours
    {
    
        public int ActualHoursId { get; set; }
        [Required]
        public int UserId { get; set; }
        [Required]
        public int ProjectId { get; set; }
        [Required]
        [Range(0, double.MaxValue, ErrorMessage = "Please enter valid double Number")]
        public double CountHours { get; set; }
        [Required]
        public DateTime date { get; set; }


    }
}
